enterprise/reporting/reporting.go

```diff
@@ -469,7 +496,10 @@ func (handle *HandleT) Report(metrics []*types.PUReportedMetric, txn *sql.Tx) {
 	reportedAt := time.Now().UTC().Unix() / 60
 	for _, metric := range metrics {
 		workspaceID := handle.getWorkspaceID(metric.ConnectionDetails.SourceID)
-		metric := transformMetricForPII(*metric, getPIIColumnsToExclude())
+		metric := *metric
+		if handle.IsPIIReportingDisabled(workspaceID) {
+			metric = transformMetricForPII(metric, getPIIColumnsToExclude())
+		}
```

Comment:
these lines appear to be not covered by tests

Reply:
Deferred writing tests for this